_data/simple-icons.json

```
@@ -9344,6 +9344,12 @@
             "source": "https://docs.pi-hole.net",
             "guidelines": "https://pi-hole.net/trademark-rules-and-brand-guidelines/"
         },
+        {
+            "title": "Piaggio Group",
+            "hex": "000000",
```

```suggestion
            "hex": "00B4DC",
```

I'd go with this, they show examples of monochrome icon against the blue - so I think we're safe to do this ourselves!

---

I linked the wrong logo guide 😔
The correct one is this: https://www.piaggiogroup.com/sites/piaggiocorpd7/files/doc/guidalogo_piaggioec_1.pdf

For full color logo the guide says:
The logo can be positioned on a background of any colour, including non-uniform and photographic
backdrops, as long as the colours are not changed and every part is visibly distinct and readable.
Next page:
In special cases the logo can be coloured with a grey tone or silver Pantone®
as long as the readability
is not affected.
Last page:
Do not invert the logo/lettering layout. Use only corporate colors and do not change their
position inside the logo.

Correct me if I'm wrong, doesn't this mean either use black/grey or the full color logo?

---

Ooo that's frustrating! The other Piaggio doc shows its monochrome logo on the coloured background, which is what I was referring to. Guess we'll have to be boring with this one 😭🙃

---

Yeahhh 🥺